packages/backend/src/open_payments/payment/outgoing/lifecycle.ts

```diff
@@ -85,8 +85,8 @@ export async function handleSending(
   const payEndTime = Date.now()
 
   if (deps.telemetry) {
-    deps.telemetry.incrementCounter('transactions_total', 1, {
-      description: 'Count of funded transactions'
+    deps.telemetry.incrementCounter('transactions_count', 1, {
+      description: 'Count of funded outgoing transactions'
```

Comment:
can you elaborate on why we're renaming `transactions_total` -> `transactions_count` (I remember the intermediate name). Just for consistency with the other incoming transaction count? Just asking since now we'll have transactions_total (unused from this point on) and now this new one that is the same thing.

I guess its all still in the test pool of data so maybe it doesnt matter that much. More specific description definitely makes sense (leave the high level "transaction" more for the display).

Reply:
Oops, I thought it was in the original issue, buty it's not. I'll revert it back to `transaction_total`